client.cpp

```diff
@@ -173,19 +181,33 @@ class Client {
         }
 
         bool files_sent = false;
-        while ((ent = readdir(dir)) != NULL) {
+        while ((ent = readdir(dir))!= NULL) {
             if (strcmp(ent->d_name, ".") == 0 || strcmp(ent->d_name, "..") == 0) {
                 continue;
             }
 
             string filepath = directory_path + string(ent->d_name);
```

Comment:
@Vansh7388 
Here, When we give the full path of the directory for example `/home/user/Desktop/test`
This line will send the full path to server right from the root directory.
And server will try to create same folder structure from the root directory path.
Instead client should only send the last leaf directory from the path here.